apps/docs/src/app/platform/component-docs/platform-forms/platform-date-picker/platform-date-picker-header/platform-date-picker-header.component.html

```diff
@@ -1,18 +1,18 @@
 <header>Date Picker</header>
 <description>
-    Platform DatePicker is an extension of the Date Picker component present in
+    Platform Date Picker is an extension of the Date Picker component present in
     <fdp-link href="fundamental-ngx#/core/datePicker">fundamental-ngx/core</fdp-link> to work with
     <fdp-link href="fundamental-ngx#/platform/form">Platform Form</fdp-link>. <br />
-    It keeps the same property and behavior with core DatePicker. <br />
-    Platform DatePicker module uses DatetimeAdapter provider for handling date parsing and formatting.
+    It keeps the same property and behavior with core Date picker. <br />
```

Comment:
Can you capitalize "Date picker" to "Date Picker" to make it consistent with the rest of the document?

Reply:
Done the changes.